cts/aggregation/stages/bucket.json

```
@@ -0,0 +1,69 @@
+{
+  "simple": {
+    "request": {
+      "aggregate": "values",
+      "pipeline": [
+        {
+          "$bucket": {
+            "groupBy": "$v",
+            "boundaries": [
+              {
+                "$numberInt": "0"
+              },
+              {
+                "$numberInt": "50"
+              },
```

```suggestion
              {
                "$numberInt": "41"
              },
```

Just so that not all numbers are in bucket zero.